pkg/core/localize/locales/en/cmd/cluster.en.toml

```diff
@@ -144,7 +144,7 @@ one = 'Connect your services to Kubernetes or OpenShift'
 one = '''
 Connect your application services to an Kubernetes or OpenShift cluster.
 
-Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/service-discovery#installing-the-rhoas-operator-on-openshift.
+Note: Before you can connect your application service to OpenShift, you need to install the RHOAS Operator in your OpenShift cluster. For installation instructions, see https://github.com/redhat-developer/app-services-guides/tree/main/docs/kafka/service-binding-kafka#installing-the-rhoas-operator-on-openshift.
 
```

Comment:
Suggest "you must install the RHOAS Operator..." as per IBM Style Guide

Reply:
@smccarthy-ie I agree with the suggested change, but I think it's outside the scope of this PR -- there are many other such references that should also be updated. I've created a new ticket to cover that work.

Reply:
Thanks, @wtrocki. Resolving this conversation.